samples/buildstrategy/buildah/buildstrategy_buildah_cr.yaml

```diff
@@ -17,9 +17,34 @@ spec:
         - |
           set -euo pipefail
 
+          echo "Creating registries config file..."
+
+          formate(){
```

Comment:
Maybe I am missing something, but why `formate` instead of `format`?

Reply:
Sorry, a spelling error,  has been corrected.